distribution/docs/src/main/resources/content/_managing/_configuring/reusing-configurations.adoc

```diff
@@ -83,6 +83,8 @@ The same step-by-step process above can be followed when migrating configuration
 *** Security STS Guest Claims Handler
 *** Security STS Server
 *** Session
+*** All Sources and their configurations
```

Comment:
✏️ I think we should specify they are "Catalog Sources". What about `All Catalog Source configurations`?